src/plugins/discover/public/embeddable/initialize_fetch.ts

```diff
@@ -181,9 +181,9 @@ export function initializeFetch({
               abortSignal: currentAbortController.signal,
               sessionId: searchSessionId,
               inspector: {
-                adapter: requestAdapter,
+                adapter: inspectorAdapters.requests,
                 title: i18n.translate('discover.embeddable.inspectorRequestDataTitle', {
-                  defaultMessage: 'Data',
+                  defaultMessage: 'Table',
```

Comment:
Do we need to remove `discover.embeddable.inspectorRequestDataTitle` from translation files?

Reply:
Updated just in case